Makefile

```diff
@@ -53,7 +53,10 @@ ios: setup
 	  -headers ${IOS_GENERATION_DIR}/headers \
 	  -library ../../target/ios-simulator/libuniffi_wysiwyg_composer.a \
 	  -headers ${IOS_GENERATION_DIR}/headers \
-	  -output ${IOS_PACKAGE_DIR}/WysiwygComposerFFI.xcframework
+	  -output ${IOS_PACKAGE_DIR}/WysiwygComposerFFI.xcframework && \
+	sed -i "" -e '1h;2,$$H;$$!d;g' -e 's/) -> ComposerUpdate {\n        return try! FfiConverterTypeComposerUpdate.lift(\n            try!/) throws -> ComposerUpdate {\n        return try FfiConverterTypeComposerUpdate.lift(\n            try/g' ${IOS_PACKAGE_DIR}/Sources/WysiwygComposer/WysiwygComposer.swift && \
+	sed -i "" -e '1h;2,$$H;$$!d;g' -e 's/) -> ComposerUpdate/) throws -> ComposerUpdate/g' ${IOS_PACKAGE_DIR}/Sources/WysiwygComposer/WysiwygComposer.swift
+
```

Comment:
I guess this could break in future if the behaviour of `uniffi-bindgen` changes? Would it be worth adding a comment in `ComposerModelWrapper.swift` to help future maintainters track down any potential issues?

And maybe a comment here to explain what it is and why it's here?

Reply:
Done